app/ts/components/pages/ConfirmTransaction.tsx

```diff
@@ -320,11 +320,54 @@ const WebsiteErrors = ({ website, websiteSocket, simulationMode }: NetworkErrorP
 	return <ErrorComponent warning = { true } text = { <> { message.message } <Link url = { message.suggestedAlternative } text = { 'Suggested alternative' } websiteSocket = { websiteSocket } /> </> }/>
 }
 
-type ModalState = 
+type ModalState =
 	{ page: 'modifyAddress', state: ModifyAddressWindowState } |
-	{ page: 'editEns', state: EditEnsNamedHashWindowState } | 
+	{ page: 'editEns', state: EditEnsNamedHashWindowState } |
 	{ page: 'noModal' }
 
+type ButtonsParams = {
+	currentPendingTransactionOrSignableMessage: PendingTransactionOrSignableMessage | undefined
+	reject: () => void
+	approve: () => void
+	confirmDisabled: boolean
+}
+function Buttons({ currentPendingTransactionOrSignableMessage, reject, approve, confirmDisabled }: ButtonsParams) {
```

Comment:
actually quite curious why `onlyReject`'s `<button>` element here is wrapped with a "flexed" `div` but it's the only child of that div?

Anyways, the `rejectOnly` component can also be moved outside this component

```ts
// 	const onlyReject = <div style = 'display: flex; flex-direction: row;'> ...
const RejectButton = ({ onClick }: { onClick: () => void }) => <div style = 'display: flex; flex-direction: row;'>
		<button className = 'button is-primary is-danger button-overflow dialog-button-left' onClick = { onClick } >
			{ 'Reject' }
		</button>
	</div>

function Buttons({ currentPendingTransactionOrSignableMessage, reject, approve, confirmDisabled }: ButtonsParams) {
//... then later
	if (identified === undefined) return <RejectButton onClick = { reject } />
```

Reply:
the flex style is just remainer when there used to be two buttons. I removed the styling here and also moved the reject out